apps/console/src/public/deployment.config.json

```diff
@@ -676,7 +676,7 @@
                 "scopes": {}
             },
             "identityProviders": {
-                "disabledFeatures": [],
+                "disabledFeatures": ["identityProviders.customAuthentication"],
```

Comment:
I assume we have added these configs in the framework side and in the deployment pipeline?